pkg/configuration/digger_config.go

```diff
@@ -50,11 +56,17 @@ type Stage struct {
 }
 
 type Workflow struct {
+	Envs          Envs                   `yaml:"envs"`
```

Comment:
Should we call it EnvVars? just to avoid confusion with any kind of "environments"